### DISCUSSION TITLE
Remotely trigger top up

### DISCUSSION THREAD

**Discussion by Qhilm** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1343] [Ideas]
From time to time I have the situation where I need to leave the house early with laptop and I remember to get my laptop to 100% when I'm in bed already. Getting downstairs to click "top up" is always annoying.

Any idea to do so remotely? Is there a CLI command that does this?

(I have the Pro version)

**Comment by 001ProMax** [comment]
You can ssh to run apple shortcuts which maintain Aldent's Shortcut action.

> From time to time I have the situation where I need to leave the house early with laptop and I remember to get my laptop to 100% when I'm in bed already. Getting downstairs to click "top up" is always annoying.
> 
> Any idea to do so remotely? Is there a CLI command that does this?
> 
> (I have the Pro version)
